### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by lokeshbv**
- Trivial code refactoring in audio and kernel modules 

- tiva: Update tiva lm3s ethernet driver 

- configs/qemu: Update qemu defconfigs

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [07978cd1529e20e9f889db52dc481a7d00d3412a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [07978cd1529e20e9f889db52dc481a7d00d3412a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a98b540d00c35d88984657a1a7e4207c31a2141d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a98b540d00c35d88984657a1a7e4207c31a2141d] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@lokeshbv I mean, ```.x%``` is not wrong expression so that it is not "fix". It is modified by our decision.
I can find ```.x%``` easily. When I send 100 packets and 15 packets are lost, 10.5% is lost. To someone, 0.5% can be meaningful (ie. for military or something else). Either not to others. We don't need to say it is meanless and not significant.

File: apps/examples/testcase/le_tc/network/itc_net_setsockopt.c [hunk 1]
```diff
@@ -44,6 +44,7 @@ static void itc_net_setsockopt_n_invalid_opt_name(int s)
 	TC_SUCCESS_RESULT();
 }
 
+#if 0
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/network/itc_net_setsockopt.c]: This is working on ARTIK and Cypress TC. Why do you comment this?

File: apps/examples/testcase/le_tc/network/itc_net_setsockopt.c [hunk 2]
```diff
@@ -121,6 +123,7 @@ static void itc_net_setsockopt_p_rcvtimo(int s)
...
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/network/itc_net_setsockopt.c]: same as previous

File: apps/examples/testcase/le_tc/network/tc_net_getsockopt.c [hunk 3]
```diff
@@ -248,7 +248,7 @@ void net_getsockopt_main(void)
 	tc_net_getsockopt_invalid_filedesc_n();
 	tc_net_getsockopt_optval_n(fd);
 	/* get has lwip_tcp in the accepticonn option and not in set. */
-//	tc_net_getsockopt_sol_socket_so_acceptconn_p(fd);
+	tc_net_getsockopt_sol_socket_so_acceptconn_p(fd);
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/network/tc_net_getsockopt.c]: Is this confirmed on ARTIK?

**Comment by seinfra** [comment]
Target : [0a59d9035e05d666f3794b2ef349a74d4cc7ca84] -

**Comment by seinfra** [comment]
Target : [0a59d9035e05d666f3794b2ef349a74d4cc7ca84] -

**Comment by lokeshbv** [comment]
@sunghan-chang, I agree with your opinion about output format type of percentage loss in ping utils.

**Comment by seinfra** [comment]
Target : [2e5b774e5009bfd7cc0977ec71c4a8ad3b99aa55] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2e5b774e5009bfd7cc0977ec71c4a8ad3b99aa55] - Code Rule Check (C++) OK.

**Pull request merged**